web/api/events.js

```diff
@@ -70,6 +70,20 @@ exports.register = function (server, eOptions, next) {
       tags: ['api', 'events'],
     },
   }, {
+    method: 'GET',
+    path: options.basePath + '/user/event/{eventId}/applications',
```

Comment:
could we use template literals

Reply:
original PR was done before migration to ES6. Will fix